tests/test_task.py

```diff
@@ -42,3 +42,23 @@ def test_task_list_multi(hq_env: HqEnv):
     ]
     assert r.get_column_value("State") == ["FINISHED"] * 10
     assert r.get_column_value("Error") == [""] * 10
+
+
+def test_task_info(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+
+    hq_env.command(["submit", "--array=5-7", "--", "bash", "-c", "hostname"])
+    wait_for_job_state(hq_env, 1, "FINISHED")
+
+    r = hq_env.command(["task", "info", "1", "5"], as_table=True)
+    assert r.get_row_value("Task ID") == "5"
+
+    r = hq_env.command(["task", "info", "1", "4-6"])
```

Comment:
I think that it shouldn't be too hard to parse multiple tables? Could you try to take a look at the implementation of `as_table` and find if we can generalize it to multiple tables? Probably with some other function, to avoid forcing all users of `as_table` to treat the return value as a list.